command/version.go

```diff
@@ -0,0 +1,16 @@
+package command
+
+import (
+	"fmt"
+	"os"
+)
+
+// CommandVersion (overridden in main.go)
+var CommandVersion string
+
+func init() {
+	Command.Version = func() {
+		fmt.Println(CommandVersion)
+		os.Exit(0)
```

Comment:
Is this needed? Won't the function just return to main.go and it will be handled there?

Reply:
Since it's just a flag, `main.go` will just keep going if we don't exit ourselves (so then you get some error messages).